### DISCUSSION TITLE
I see no commits to from Kaveh during the year

### DISCUSSION THREAD

**Discussion by svetlyak40wt** [kaveh808/kons-9#263] [General]
Hi @kaveh808 !

Are you ok? Is everything all right in your life?

**Comment by kaveh808** [comment]
Hello Alexander,

I am doing well.  Started a new job early this year which has been keeping me busy. Teaching at the University of Hawaii.

Still have plans to return to kons-9 development, but to be honest I'm not sure what the next steps should be. I don't have a good handle on who is using the code and for what ends.  Whether I should add more OpenGL features (lights, materials, shaders), add support for a renderer, or develop more research-y code (I'm a big fan of genetic algorithms and genetic programming).

Thank you for reaching out. I do appreciate the concern. :)

Regards,

Kaveh

**Comment by svetlyak40wt** [comment]
I'm glad to hear you are doing well. Hope you will be able to work on this cool project some day. Seems it just need a product manager who will create a roadmap.

**Comment by foretspaisibles** [comment]
I was also kind of wondering! :-) Should we organise some online workshop to draft a roadmap — it could be easier and nicer to have a conversation!

I believe that KONS9 has an enormous potential, maybe in a workshop we could figure out a few use-cases and see how to prioritise them? Congratulations on the work done so far!

**Reply by kaveh808** [reply]
That sounds like an attractive idea. How best to organize it?
